CWE/localisation/Chrisfox's events.csv

```diff
@@ -548,3 +548,68 @@ EVTDESC8120711;Tatarstan continues to work under the leadership of the Supreme S
+NNG_nationalist_dictatorship_ADJ;Hausa;;x;;;;;;;;;;;;;;;;
+NNG_populist_dictatorship_ADJ;Hausa;;x;;;;;;;;;;;;;;;;
+NNG_absolute_monarchy;Sokoto;;x;;;;;;;;;;;;;;;;
+NNG_absolute_monarchy_ADJ;Sokoto;;x;;;;;;;;;;;;;;;;
+NNG_hms_government;Hausa Kingdom;;x;;;;;;;;;;;;;;;;
+NNG_hms_government1;Hausa Kingdom;;x;;;;;;;;;;;;;;;;
+NNG_hms_government_ADJ;Hausa;;x;;;;;;;;;;;;;;;;
+NNG_hms_government1_ADJ;Hausa;;x;;;;;;;;;;;;;;;;
+democratic_bloc_desc;This nation is aligned with the 'Democratic Bloc', pivoting its foreign policy towards the nation of the United States and its allies and away from the Soviet Union and its own sphere.;;x;;;;;;;;;;;;;;;;
+communist_bloc_desc;This nation is aligned with the 'Communist Bloc', pivoting its foreign policy towards the Union of Soviet Socialist Republics and its allies and away from the United States and its own sphere.;;x;;;;;;;;;;;;;;;;
+military_loyalty_high_desc;We are experiencing a high rate of approval from the military, increasing their morale and decreasing the likelihood of any potential troubles with disgruntled generals and soldiers.;;x;;;;;;;;;;;;;;;;
+military_loyalty_medium_desc;Our military currently has an average approval for our current regime and policies, making a coup unlikely.;;x;;;;;;;;;;;;;;;;
+military_loyalty_low_desc;The military is currently plagued with low approval for the current regime, with open displays of dissent from soldiers and generals alike. Its highly probable that morale will suffer as a result, and that continued unrest within the ranks of the armed forces could lead to worse consequences.;;x;;;;;;;;;;;;;;;;
+minimal_societal_corruption_desc;Government corruption is barely noticable in $COUNTRY$, with administration and bureaucracy operating efficiently within the confines of our established laws.;;x;;;;;;;;;;;;;;;;
+low_societal_corruption_desc;Government corruption in $COUNTRY$ is fairly low, with rates of bribery and deception from administrators and bureaucrats far and few between.;;x;;;;;;;;;;;;;;;;
+medium_societal_corruption_desc;Corruption in $COUNTRY$ is starting to become a problem, with government efficiency starting to suffer as a result.;;x;;;;;;;;;;;;;;;;
+large_societal_corruption_desc;With a large-scale problem with government corruption, $COUNTRY$ faces several issues in effective governance, as well as establishing its legitimacy to the people.;;x;;;;;;;;;;;;;;;;
+massive_societal_corruption_desc;Corruption is rife within the government of $COUNTRY$, with taxation, administration, education, and most other institutions struggling to work at all amid open bribery and near constant scandals. Without immediate action, the problem is likely to cause economic and social unrest.;;x;;;;;;;;;;;;;;;;
+promoting_terrorists_desc;In a move that is likely to draw international condemnation, $COUNTRY$ has decided to actively foster, fund, train, and arm groups designated as 'terrorists' by the international community.;;x;;;;;;;;;;;;;;;;
```

Comment:
The way to solve this conflicting files error is to put localisation from line 578 into a new csv file.

Reply:
Thanks. It is annoying to get the error and it to just draw orange arrows everywhere.